src/utils/get-organized-arguments.test.ts

```diff
@@ -743,4 +757,38 @@ describe('#getOrganizedArguments()', () => {
 				data: 'b1',
 			});
 	});
+
+	it('Complains when data accepts does not resolve to array', async () => {
+		process.argv = [
+			'/path/to/node',
+			path.join(
+				path.normalize(path.join(__dirname, '..', '..', 'test-file-trees')),
+				'bad-structure',
+				'cli',
+				'entry.js'
+			),
+			'invalid-data-accepts',
+			'--test',
+			'hello',
+		];
+
+		await expect(getOrganizedArguments()).rejects.toThrow('Error: data.accepts must resolve to an array');
+	});
+
+	it('Complains when option accepts does not resolve to array', async () => {
+		process.argv = [
+			'/path/to/node',
+			path.join(
+				path.normalize(path.join(__dirname, '..', '..', 'test-file-trees')),
+				'bad-structure',
+				'cli',
+				'entry.js'
+			),
+			'invalid-option-accepts',
+			'--test',
+			'hello',
+		];
+
+		await expect(getOrganizedArguments()).rejects.toThrow("option['test'].accepts must resolve to an array");
```

Comment:
Specifically, this test does not have the "Error: " prefix like the test above. Why is that?

Reply:
See [this response](https://github.com/sparksuite/waterfall-cli/pull/373#discussion_r681192705) for the reason.